src/test/java/APITest/Payload.java

```diff
@@ -0,0 +1,53 @@
+package APITest;
+
+public class Payload {
+	  
+	// 1.. register successful
+	public static String register()
+		{
```

Comment:
It is important to indent the code consistently as it makes it much more easy to read/write/modify the code. Most IDEs have options/shortcuts to auto indent the code. Please find a way to do it for your IDE so that you don't have to do the indentation manually.